README.md

```diff
@@ -15,12 +15,18 @@ in `.targets/datasource/scala-2.12/quasar-s3-<version>-explode.tar.gz`
 
 ## Configuration
 
-You can create a new S3 datasource after you've loaded the plugin into
+You can create a new S3 datasource after you've loaded this plugin into
 Quasar. Refer to the previous section for instructions on how to do
-that. In order to create a datasource, you will need to send a PUT
-request to `/datasource/<your datasource name>` including a JSON
-document specifiying the datasource's configuration. An example of a
-JSON configuration to create a datasource that parses line-delimited JSON:
+that. In order to create a datasource, you will need to send a POST
+request to `/datasource` including a JSON
+document specifiying the datasource's configuration. The format of the
+JSON document can be found in [`slamdata-backend`'s
+documentation.](https://github.com/slamdata/slamdata-backend#applicationvndslamdatadatasource). 
```

Comment:
Makes a reference to a private repository (from a public repos). Also the first section assumes `slamdata-backend`. Not sure if this is the best place for this documentation. 

Just to keep in mind, I think quasar/repl should be mentioned too in the end, but we can do this once the README of quasar is in place.